src/components/memorygame/changeCardStatePropertyToOpposite.jsx

```diff
@@ -0,0 +1,48 @@
+// manipulates isFlipped and isMatched property of card a state
```

Comment:
you dont want to put functions one per file like this. memorygame/gameHelpers.jsx

Reply:
PTAL